flask_project/campaign_manager/views.py

```diff
@@ -56,37 +65,99 @@
 
 MAX_AREA_SIZE = 320000000
 
+# create session for Database Queries
+Session = sessionmaker(bind=engine)
+Session.configure(bind=engine)
+session = Session()
+session.rollback()
+
+
+@campaign_manager.route('/add_osm_user/<username>', methods=['GET', 'POST'])
+def add_osm_user(username):
+    user = session.query(User).filter(User.osm_user_id == username).all()
```

Comment:
I dont think you need to execute this query and then check if it returns an user.
Value for the Column osm_user_id must be unique (models.py:124)
So you could just try to create a new User and check if it fails or not.
pseudocode:
```
new_user  = User(osm_user_id=username)
session.add(new_user)
if session.commit():
   return 201 (OK, created)
else
  return 422 (Unprocessable entity or something else)
```